docs/how-to-guides/environment-setup-and-maintenance/setup-isis-data.md

```diff
@@ -0,0 +1,153 @@
+
+## SPK kernels
+
+SPK stands for Spacecraft Position Kernels. They contain the ephemeris (position and velocity) information for spacecraft, planetary bodies, etc. The kernels usually have the ``.bsp`` extension. Create the directory:
+
+```sh
+
+  $ISISDATA/chandrayaan2/kernels/spk
+
+```
+
+and copy there these files. Run a command as:
+
+```sh
+
+   cd ${ISISDATA}/chandrayaan2/kernels/spk
+
+   kerneldbgen                                       \
+    to = '$chandrayaan2/kernels/spk/kernels.????.db' \
+    type = SPK                                       \
+    recondir = '$chandrayaan2/kernels/spk'           \
+    reconfilter = 'ch2*.bsp'                         \
+    lsk = '$base/kernels/lsk/naif????.tls'
+
+```
+
+It is very important to use single quotes above, not double quotes, so that the shell does not expand these variables. This also ensures relative paths are created, rather than absolute ones specific to a given file system.
+
+This will create an index file such as ``kernels.0000.db``. Before rerunning this command, delete any existing ``.db`` files, as otherwise new entries will be made.
+
+Save this command to a shell script in the current directory named ``makedb``, so that it keeps a record of how the index was produced. See also the ``makedb`` script for other datasets, for comparison.
+
+## CK kernels
+
+ CK (Spacecraft Pointing Kernels) contain the attitude (orientation) information for the spacecraft and its instruments. The command for assembling the index file for this is very similar.
+
+```sh
+
+   cd ${ISISDATA}/chandrayaan2/kernels/ck
+
+   kerneldbgen                                           \
+     to = '$chandrayaan2/kernels/ck/kernels.????.db'     \
+     type = CK                                           \
+     recondir = '$chandrayaan2/kernels/ck'               \
+     reconfilter = 'ch2*.bc'                             \
+     sclk = '$chandrayaan2/kernels/sclk/ch2_sclk_v1.tsc' \
+     lsk = '$base/kernels/lsk/naif????.tls'
+```
+
+## IK Kernels
+
+IK (Instrument Kernels) provide detailed information about the instrument's physical properties, such as focal length, pixel scale, and lens distortion models.
+
+Go to ``$ISISDATA/chandrayaan2/kernels/ik``. Create a file named ``kernels.0000.db`` with content along the lines of:
+
+```sh
+
+   Object = Instrument
+     Group = Selection
+       Match = ("Instrument", "InstrumentId","IIR")
+       File  = ("chandrayaan2", "kernels/ik/ch2_iir_v01.ti")
+     EndGroup
+
+     Group = Selection
+       Match = ("Instrument", "InstrumentId","OHRC")
+       File  = ("chandrayaan2", "kernels/ik/ch2_ohr_v01.ti")
+     EndGroup
+
+     Group = Selection
+       Match = ("Instrument", "InstrumentId","TMC-2")
+       File  = ("chandrayaan2", "kernels/ik/ch2_tmc_v01.ti")
+     EndGroup
+   EndObject
+
+```
+
+It is very important to have the precise name of each instrument, and to ensure the proper ``.ti`` files are passed in.
```

Comment:
Probably worth mentioning where you get the instrument IDs from. Maybe put this after FK?

Reply:
Order swapped. I don't know where to get instrument id from. The FK dir does not have that info.